maintainers/this-month-in-nix-docs/make-post.sh

```diff
@@ -57,6 +57,9 @@ The Documentation Team can be found in a number of places. Check the [Documentat
 
 # AUTOGENERATED
 
+# How You Can Help
```

Comment:
I'd put this as a `##` between `Projects` and `PRs`. Right now this would be the only `#` in the entire (rendered) document and it's somewhere in the middle.

Reply:
```suggestion
...
cat << EOF
# This Month in Nix Docs - #NUMBER - MONTH YEAR

## Community
The Documentation Team can be found in a number of places. Check the [Documentation Team page](https://nixos.org/community/teams/documentation.html) and>

## Projects

## How You Can Help
If you like what we're doing, consider [joining the documentation team](https://nixos.org/community/teams/documentation) or [donating to the NixOS Found>

## RFCs

## Documentation PRs Merged

# AUTOGENERATED
...
```

You mean something like this? Then in the rendered document you get:

```
# This Month in Nix Docs - #NUMBER - MONTH YEAR

## Community
The Documentation Team can be found in a number of places. Check the [Documentation Team page](https://nixos.org/community/teams/documentation.html) and>

## Projects

## How You Can Help
If you like what we're doing, consider [joining the documentation team](https://nixos.org/community/teams/documentation) or [donating to the NixOS Found>

## RFCs

## Documentation PRs Merged

# AUTOGENERATED

## PRs

### NixOS/nix
- [#8788](https://github.com/NixOS/nix/pull/8788) Port the flags of nix-daemon to nix daemon ([@bryanhonof](https://github.com/bryanhonof))
- [#8774](https://github.com/NixOS/nix/pull/8774) Add infra for experimental store implementations ([@Ericson2314](https://github.com/Ericson2314))
...
```

In the final post, you add some discussion in [Projects](https://discourse.nixos.org/t/this-month-in-nix-docs-4-july-august-2023/32502#projects-2) above the PRs merged, so this way the call to action happens right after.

Reply:
You mean something like this?
```diff
cat << EOF
# This Month in Nix Docs - #NUMBER - MONTH YEAR

## Community
The Documentation Team can be found in a number of places. Check the [Documentation Team page](https://nixos.org/community/teams/documentation.html) and drop in if you'd like to contribute!

## Projects

## How You Can Help
If you like what we're doing, consider [joining the documentation team](https://nixos.org/community/teams/documentation) or [donating to the NixOS Foundation's documentation project on Open Collective](https://opencollective.com/nixos/projects/documentation-project) to fund ongoing maintenance and development of learning materials and other documentation.

## RFCs

## Documentation PRs Merged

# AUTOGENERATED

EOF
```

Then in the rendered document you get something like this:

```shell

## Community
The Documentation Team can be found in a number of places. Check the [Documentation Team page](https://nixos.org/community/teams/documentation.html) and drop in if you'd like to contribute!

## Projects

## How You Can Help
If you like what we're doing, consider [joining the documentation team](https://nixos.org/community/teams/documentation) or [donating to the NixOS Foundation's documentation project on Open Collective](https://opencollective.com/nixos/projects/documentation-project) to fund ongoing maintenance and development of learning materials and other documentation.

## RFCs

## Documentation PRs Merged

# AUTOGENERATED
```

In the Discourse post, you add some commentary in the [Projects](https://discourse.nixos.org/t/this-month-in-nix-docs-4-july-august-2023/32502#projects-2) section, so this makes the call to action appear right after that.